src/docfx/template/MustacheTemplate.cs

```diff
@@ -44,13 +47,16 @@ private static RendererSettingsBuilder UseJson(RendererSettingsBuilder settings,
         {
             // JObject implements IEnumerable, stubble treats IEnumerable as array,
             // need to put it to section blacklist and overwrite the truthy check method.
-            var sectionBlacklist = RendererSettingsDefaults.DefaultSectionBlacklistTypes();
-            sectionBlacklist.Add(typeof(JObject));
-
             return settings.AddValueGetter(typeof(JObject), GetJObjectValue)
+                           .AddValueGetter(typeof(JArray), GetJArrayValue)
+                           .AddValueGetter(typeof(IList), OverwriteValuleGetter) // overwrite default value getters in stubble
```

Comment:
refer to https://github.com/StubbleOrg/Stubble/blob/master/src/Stubble.Core/Settings/RendererSettingsDefaults.cs